addons/l10n_es/views/report_invoice.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <!-- add company id -->
+        <xpath expr="//div[@id='partner_vat_address_not_same_as_shipping']" position="after">
+            <div t-if="o.partner_id.company_registry">
+                ID: <span t-field="o.partner_id.company_registry"/>
```

Comment:
Is there a moment where this `ID:` should be translated? (I know commonly, it does not really need it, but what for Chinese people for ex) 
If so, it should be in a .pot.

Reply:
Since it's change for the spanish localisation i thought that it was not really necessary but i can add it to the pot to be sure yes :+1:

Reply:
If I'm not mistaken, we sometimes translate this invoice if the client has another language no? That's what I was thinking about

Reply:
I don't remember exactly if the invoice is translate according to the language of the client but anyway i will add the entry in the pot you are right thanks !